src/Components/Search.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { FcSearch } from "react-icons/fc";
+const Search = () => {
+  return (
+    <div className="flex items-center mt-5 w-full md:w-full h-10">
+        <FcSearch className='absolute top-22 left-19 z-10 mr-2'/>
```

Comment:
increase the negative margin for the search icon to make it look like it is no on the edge of the bar